lib/msal-browser/test/crypto/CryptoOps.spec.ts

```diff
@@ -4,30 +4,46 @@ import { BrowserCrypto } from "../../src/crypto/BrowserCrypto";
 import { createHash } from "crypto";
 import { PkceCodes, BaseAuthRequest } from "@azure/msal-common";
 import { TEST_URIS } from "../utils/StringConstants";
-import { DatabaseStorage } from "../../src/cache/DatabaseStorage";
-import { BrowserAuthError, BrowserAuthErrorMessage } from "../../src";
+import { BrowserAuthError } from "../../src";
+
 const msrCrypto = require("../polyfills/msrcrypto.min");
 
+let mockDatabase = {
+    asymmetricKeys: {},
+    symmetricKeys: {}
+};
+
+// Mock DatabaseStorage
+jest.mock("../../src/cache/DatabaseStorage", () => {
+    return {
+        DatabaseStorage: jest.fn().mockImplementation((tableName: string) => {
```

Comment:
This is great. Looks like its used in a few places, possible to factor out into a helper?

Reply:
I can't figure out how to keep the `mockDatabase` object global without having the mocked `DatabaseStorage` object in the same test file. I've also been wondering why we have things like the mocked crypto interface repeated across a bunch of browser tests, I assume it's a similar problem.